packages/boot/test/bootstrapTests/test-orchestration.ts

```diff
@@ -167,9 +169,131 @@ test.serial('stakeAtom - smart wallet', async t => {
   t.like(wd.getLatestUpdateRecord(), {
     status: { id: 'request-account', numWantsSatisfied: 1 },
   });
-
-  const { ATOM } = agoricNamesRemotes.brand;
+  const { ATOM, DAI_axl, BLD } = agoricNamesRemotes.brand;
   ATOM || Fail`ATOM missing from agoricNames`;
+  DAI_axl || Fail`DAI_axl missing from agoricNames`;
+  BLD || Fail`BLD missing from agoricNames`;
+
+  // 2. Deposit to Account
+  await wd.executeOffer({
+    id: 'request-deposit-success',
```

Comment:
success is independent of the offer identity
```suggestion
    id: 'request-deposit',
```

reading further I see you have other variants. Still I think success or failure are independent of identity. E.g.

```
request-deposit
request-deposit-bad-want
request-deposit-two-gives
```

Reply:
I tripped over that name/id on 1st reading too.